tests/conftest.py

```diff
@@ -38,3 +40,14 @@ def samesite(request, settings):
 def samesite_invalid(settings):
     """Set samesite flag to strict."""
     settings.SESSION_COOKIE_SAMESITE = "Hello"
+
+
+@pytest.fixture(autouse=True)
```

Comment:
I'm not keen on the `autouse`. I can't see just from the code whether or when it's in play.

Reply:
This one is for rolling back the changes by `sys.module`.

> And then, should I use files instead of sys.modules?
> 
> @jjjkkkjjj hold off on that last. If you could add the drafts for the docs (don't have to be perfect) I can think it through properly

I ask you again.
Should I create the test files instead of using `sys.module`?